float64 x
float64 y
float64 z
float64 xkat
float64 ykat
float64 zkat
float64 time
string type
---
string status
